src/components/Menu/SideMenu/index.js

```diff
@@ -72,6 +72,7 @@ export class SideMenu extends PureComponent /*:: <Props, State> */ {
         <div role="menu">
           {this.props.visible && this.props.showConnectionStatusToast ? (
             <Tip
+              title="💡 How strong are the connections?"
               body="The green and red signals of the connection status shows the connectivity strength of the URLs"
```

Comment:
The green signal is displayed when an external dependency is available whist a red signal indicates that InterPro cannot connect to the resource.